lor_deckcodes/decode.py

```diff
@@ -27,10 +35,23 @@ def decode_deck(deckcode: str):
 
     # 3 card copies
     all_cards.extend(_decode_card_block(3, data))
+
     # 2 card copies
     all_cards.extend(_decode_card_block(2, data))
+
     # 1 card copies
     all_cards.extend(_decode_card_block(1, data))
+
+    # more cards
+    n = data.tell()
```

Comment:
Please use more relevant variable names, the code gets very hard to read with single letter names